README.md

```diff
@@ -69,8 +69,9 @@ This is my first mobile version port folio microverse project
 <!-- LIVE DEMO -->
 
 ## 🚀 Live Demo <a name="live-demo"></a>
-
 [live Demo](https://osmanwako.github.io/Desktoportfolio)
+<!-- GETTING STARTED -->
+
 
 ## 💻 Getting Started <a name="getting-started"></a>
```

Comment:
- [ ] Please always ensure that you have a professional pull request. This pull request does not have a summary/description at all and this is against best practice. Also, I did not see any review comment from previous reviewers, I am wondering if you deleted it or what?

![image](https://user-images.githubusercontent.com/84156714/221804352-ccbe68f8-31df-4592-820c-9d504f148791.png)